src/Blazored.LocalStorage/LocalStorageService.cs

```diff
@@ -73,7 +78,7 @@ public void SetItem(string key, object data)
             if (e.Cancel)
                 return;
 
-            _jSInProcessRuntime.Invoke<object>("localStorage.setItem", key, JsonSerializer.Serialize(data));
+            _jSInProcessRuntime.Invoke<object>("localStorage.setItem", key, JsonSerializer.Serialize(data, this._jsonOptions));

```

Comment:
Could we drop the `this` keyword.
```suggestion
            _jSInProcessRuntime.Invoke<object>("localStorage.setItem", key, JsonSerializer.Serialize(data, _jsonOptions));
```